app/experimenter/experiments/management/commands/generate_dashboards.py

```diff
@@ -0,0 +1,53 @@
+import logging
+from django.conf import settings
+
+from redash_client.client import RedashClient
+from redash_client.dashboards.StatisticalDashboard import (
+    StatisticalDashboard)
+
+from django.core.management.base import BaseCommand
```

Comment:
Import grouping should go:

1) Python imports

2) Library imports

3) Project imports

so just add a new line after logging and move this line up with the other django import